security/pfSense-pkg-tinc/files/usr/local/www/status_tinc.php

```diff
@@ -56,8 +63,15 @@ function tinc_status_usr2() {
 	usleep(500000);
 	$clog_path = "/usr/local/sbin/clog";
 	$result = array();
+	$logfile = "/var/log/tinc.log";
+	$firmware = host_firmware_version();
 
-	exec("{$clog_path} /var/log/tinc.log | sed -e 's/.*tinc\[.*\]: //'",$result);
+	if ($firmware['config_version'] >= 19.7) {
+		exec(system_log_get_cat() . ' ' . $logfile . "| /usr/bin/sed -e 's/.*tinc\[.*\]: //'", $result);
+	} else {
+		exec("{$clog_path} /var/log/tinc.log | /usr/bin/sed -e 's/.*tinc\[.*\]: //'", $result);
+	}
+	
```

Comment:
See comments above which also apply to this similar block.
(And since they are practically identical, I have to wonder if this could be simplified significantly, though that's likely out of scope for this particular PR)